apps/cookbook/src/app/showcase/segmented-control-showcase/segmented-control-showcase.component.ts

```diff
@@ -22,15 +22,17 @@ export class SegmentedControlShowcaseComponent {
     },
     {
       name: 'items',
-      description:
-        'An array of `SegmentItem[]` representing the set of segment elements within the control.',
+      description: `An array of SegmentItem[] representing the set of segment elements within the control. If you want a badge on the SegmentItem it can be added via the SegmentItem array. *
```

Comment:
I think it would be neat to collect all the documentation regarding the use of badge under it's own "Using default with badge" headline (or similar). This would make it possible to shorten the description in the API table and not have to jump around for the footnote 😄

We could also consider removing badges from the top most example and then have one with badges under that heading.